js/actions/appActions.js

```diff
@@ -1394,6 +1394,23 @@ const appActions = {
   },
 
   /**
+   * Indicates URL bar selected index
+   *
+   * @param {number} windowId - the window ID
+   * @param {number} selectedIndex - The index for the selected item (users can select items with down arrow on their keyboard)
+   */
+  urlBarSelectedIndexChanged: function (windowId, selectedIndex) {
```

Comment:
I like this change to pass the index instead of calculating the selections in the component